pkg/serviceprovider/github/githubclientbuilder.go

```diff
@@ -40,6 +43,10 @@ func (g *githubClientBuilder) createAuthenticatedGhClient(ctx context.Context, s
 		lg.Error(tsErr, "failed to get token from storage for", "token", spiToken)
 		return nil, fmt.Errorf("failed to get token from storage for %s/%s: %w", spiToken.Namespace, spiToken.Name, tsErr)
 	}
+	if tokenData == nil {
+		lg.Error(accessTokenNotFoundError, "token data not found", "token-name", spiToken.Name, "tokenData", tokenData)
```

Comment:
I understand that the tokenData here is always logged as "nil" but it scares me that we might expose secret data if we accidentally remove the if statement (maybe I'm paranoid :)). Plus, if the data is always logged as "nil" is it needed to log this value at all?